server.js

```diff
@@ -1,16 +1,9 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
+// import doItBeFartin from "./data/doitbefartin.json";
```

Comment:
Ahh so its not needed to import since you only have one dataset in your data file? I was wondering a bit since commented out.